tests/tests_pytorch/loops/test_training_loop_flow_scalar.py

```diff
@@ -123,62 +118,6 @@ def backward(self, loss):
 
     # make sure correct steps were called
     assert model.training_step_called
-    assert not model.training_step_end_called
```

Comment:
GitHub shows these lines as deleted but that's just an illusion because they are duplicated in the test below and so the diff is just shifted.